### PR TITLE
Support for FDW

### PR DESCRIPTION
**Pull request by frankgh**
This PR adds support for FDW:
- The first commit is code clean up in the fragmenter call. This is a backwards compatible change
- The second commit adds new profiles for csv format (which point to the same Fragmenter/Accessor/Resolver as their text counterpart).

### CODE REVIEW

File: server/pxf-service/src/main/java/org/greenplum/pxf/service/rest/FragmenterResource.java [hunk 1]
```diff
@@ -86,24 +86,23 @@ public FragmenterResource() {
...
             throws Throwable {
 
         long startTime = System.currentTimeMillis();
-        LOG.debug("FRAGMENTER started for path \"{}\"", path);
-
         final RequestContext context = parseRequest(headers);
+        final String path = context.getDataSource();
         final String fragmenterCacheKey = getFragmenterCacheKey(context);
 
+        LOG.debug("FRAGMENTER started for path \"{}\"", path);
```
Comment (denalex) [inline server/pxf-service/src/main/java/org/greenplum/pxf/service/rest/FragmenterResource.java]: the original placement of the log statement helped to debug arrival of request to PXF before any substantial logic could be called. We should preserve it, perhaps log a different statement

File: server/pxf-service/src/main/resources/pxf-profiles-default.xml [hunk 2]
```diff
@@ -220,7 +220,18 @@ under the License.
     <profile>
         <name>hdfs:text</name>
         <description>This profile is suitable for using when reading delimited single line records
-            from plain text files on HDFS.
+            from plain text, tab-delimited, files on HDFS.
```
Comment (denalex) [inline server/pxf-service/src/main/resources/pxf-profiles-default.xml]: why tab-delimited ?

Comment (frankgh) [reply server/pxf-service/src/main/resources/pxf-profiles-default.xml]: This is making reference to tab-delimited text, which is the default for 'TEXT' format in the external table framework: https://gpdb.docs.pivotal.io/5190/ref_guide/sql_commands/CREATE_EXTERNAL_TABLE.html

File: server/pxf-service/src/main/java/org/greenplum/pxf/service/rest/FragmenterResource.java (hunk 1 above)
Comment (frankgh) [reply server/pxf-service/src/main/java/org/greenplum/pxf/service/rest/FragmenterResource.java]: added `Received FRAGMENTER call`

**Pull request merged**